src/components/Navigation/PageMenu/PageMenu.types.ts

```diff
@@ -0,0 +1 @@
+export type PageMenuVariants = 'buttons' | 'border';
```

Comment:
Maybe it makes more sense to make `border` a boolean prop instead of using variants

Reply:
Yeah maybe, the idea was to support having more variants in the future